elastalert/alerts.py

```diff
@@ -34,7 +35,8 @@ def _add_custom_alert_text(self):
         alert_text = self.rule.get('alert_text', '')
         if 'alert_text_args' in self.rule:
             alert_text_args = self.rule.get('alert_text_args')
```

Comment:
This is in an if block to make sure it exists. Though someone could do alert_text_args: 5 or something dumb if it's not in the JSON schema